zqd/search/ndjson.go

```diff
@@ -0,0 +1,65 @@
+package search
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/brimsec/zq/pkg/bufwriter"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio"
+	"github.com/brimsec/zq/zio/ndjsonio"
+)
+
+// NDJSONOutput implements the Output inteface and writes NDJSON encoded-output
+// directly to the client as text/NDJSON.
```

Comment:
is `text/NDJSON` what Go figures out via `http.DetectContentType`And should we explicitly set the content type as `application/x-ndjson`?

Reply:
Oops never mind I hadn't seen the `ContentType()` method below